dcs/src/main/java/org/trafodion/dcs/server/ServerManager.java

```diff
@@ -367,6 +335,36 @@ public ServerManager(Configuration conf, ZkClient zkc,
                         Constants.DEFAULT_DCS_SERVER_USER_PROGRAM_RESTART_HANDLER_RETRY_INTERVAL_MILLIS);
         this.retryCounterFactory = new RetryCounterFactory(
                 this.maxRestartAttempts, this.retryIntervalMillis);
+        serverHandlers = new ServerHandler[this.childServers];
+    }
+
+    private static boolean isTrafodionRunning(String nid) {
+
+        // Check if the given Node is up and running
+        // return true else return false.
+        // Invoke sqcheck -n <nid> to check Node status. 
+        // 
+        //   sqcheck returns:
+        //    -1 - Not up ($?=255) or node down
+        //     0 - Fully up and operational or node up
+        //     1 - Partially up and operational
```

Comment:
This is the old comment that got moved here with changes to include node down and node up.  I will check and update this comment when I merge something else next time